sdk/core/azure-core/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Renamed `NoRevoke` to `EnableCertificateRevocationListCheck` for `Azure::Core::Http::CurlTransportSSLOptions`.
 - Renamed `GetString()` to `ToString()` in `Azure::Core::DateTime`.
 - Renamed `GetUuidString()` to `ToString()` in `Azure::Core::Uuid`.
+- Moved `Version` from `Azure::Core::Details` to `Azure::Core`.
```

Comment:
I don't think we need a changelog entry for this.
To the user, the class has just appeared - whether out of Details namespace, or as brand new, should not matter. But even then, we don't have to announce all new classes. But if we do, let's put it new new features, "Added Azure::Core::Version class".

Second, do we want this for all packages? Identity, Storage?

Reply:
Agreed, not a breaking change.